site/content/support.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Where to Go for Support"
+draft: false
+weight: 1100
+toc: true
+tags: [ "docs" ]
+docs: "DOCS-OOO"
+---
+
+## Support Policy
+
+NGINX Gateway Fabric adheres to the support policy detailed at [nginx.com/support](https://nginx.com/support).
+
+## Commercial Support
+
+For paid customers of NGINX Gateway Fabric, commercial support is available by opening a support case at the [MyF5](https://my.f5.com/) portal.
+
+Generate a Support Package
+To generate a support package, run the following command:
+
+- (TBD)
+
+## Community Support
+
+For NGINX Gateway Fabric support or issues not addressed by documentation, you can reach out using the following two options:
```

Comment:
perhaps we can shorter this section to 

> If you experience issues with NGINX Gateway Fabric, please [open a bug](https://github.com/nginxinc/nginx-gateway-fabric/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=) in GitHub. If you have any suggestions or enhancement requests, please [open an idea](https://github.com/nginxinc/nginx-gateway-fabric/discussions/categories/ideas) on GitHub discussions. You can contact us directly via [kubernetes@nginx.com](mailto:kubernetes@nginx.com) or on the [NGINX Community Slack](https://nginxcommunity.slack.com/channels/nginx-gateway-fabric) in the #nginx-gateway-fabric channel.

note: this comes from https://github.com/nginxinc/nginx-gateway-fabric#contacts

note that when opening a bug, the user will see a template which describes guidelines on what info to provide -- please see https://github.com/nginxinc/nginx-gateway-fabric/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=

Reply:
updated via commit